deny.toml

```diff
@@ -20,6 +20,7 @@ allow = [
 ]
 exceptions = [
     { name = "arrayref", allow = ["BSD-2-Clause"] },
+    { name = "curve25519-dalek", allow = ["BSD-3-Clause"] },
```

Comment:
Is there a reason why we don't just put {BSD-2-Clause, BSD-3-Clause, ISC, CC0-1.0} in `allow`? Those are permissive licenses that I think are universally accepted to be compatible with Apache-2.0 and MIT.

Reply:
I'm taking the same approach here that we do in `librustzcash` and (effectively) `zcashd`. If we want to take a different approach, we should do so in separate PRs everywhere.